pyproject.toml

```diff
@@ -90,16 +90,32 @@ vulture = ">=1.0"
 jinja2 = "<3.1.0"
 
 [tool.coverage.paths]
-source = ["src", "*/site-packages"]
+source = [
+    "isort/",
+    ".tox/*/lib/python*/site-packages/isort/",
+    ".tox/*/lib/site-packages/isort/"
+]
 tests = ["tests", "*/tests"]
 
 [tool.coverage.run]
 branch = true
-source = ["git_portfolio", "tests"]
+source = ["isort", "tests"]
+omit = [
+    "isort/_vendored/*",
+    "tests/*",
+    "isort/deprecated/*",
+]
 
 [tool.coverage.report]
 show_missing = true
 fail_under = 100
```

Comment:
```suggestion
fail_under = 99
```

Reply:
@SauravMaheshkar I checked also that it was not failing before with old config but coverage was before 99.